bucket/dotnet-sdk.json

```
@@ -14,6 +14,8 @@
         }
     },
     "bin": "dotnet.exe",
+    "depends": "extras/vcredist2015",
```

Vcredists needs to be only suggested

```suggestion
    "suggest": {
        "Visual C++ Redistributable 2015": "extras/vcredist2015"
    },
```

---

Cool, was thinking that might be the recommended way to do it. 👍